data/strings/types_strings.txt

```diff
@@ -29855,6 +29855,41 @@
   [type.tourism.attraction.specified]
     ref = type.tourism.attraction
 
+  [type.tourism.camp_pitch]
+    en = Camp Pitch
+    en-GB = Camp Pitch
```

Comment:
No need to type equal translations for sublanguages. 

Some translations are missing, how did you translate it? Not with tools/python/translate.py, right?

```
    en = Camp site
    af = Kampplek
    ar = موقع المخيم
    az = Düşərgə yeri
    be = Пляцоўка для лагера
    bg = Къмпинг
    ca = Càmping
    cs = Tábořiště
    da = Lejrplads
    de = Zeltplatz
    el = Χώρος κατασκήνωσης
    es = Camping
    et = Laagriplats
    eu = Kanpalekua
    fa = سایت کمپ
    fi = Leirintäalue
    fr = Camping
    he = אתר מחנה
    hi = शिविर स्थल
    hu = Táborhely
    id = Lokasi perkemahan
    it = Campeggio
    ja = キャンプサイト
    ko = 캠프 사이트
    lt = Stovyklavietė
    mr = कॅम्प साइट
    nb = Leirplass
    nl = Camping
    pl = Kemping
    pt = Acampamento
    pt-BR = Local do acampamento
    ro = Locul de campare
    ru = Площадка для лагеря
    sk = Táborisko
    sv = Campingplats
    sw = Tovuti ya kambi
    th = เว็บไซต์ค่าย
    tr = Kamp alanı
    uk = Майданчик для таборування
    vi = Khu cắm trại
    zh-Hans = 营地
    zh-Hant = 營地
```

Reply:
I definitely can't claim this much language knowledge. This was just done manually via Google Translate, which I now realize is basically what https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md has automated tools for. Will revise with next force push.